### PR TITLE
finetuning-scheduler recipe

### PR DESCRIPTION
**Pull request by speediedan**
Checklist complete. Thanks!!
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by speediedan** [comment]
@conda-forge/help-python ready for review. Thanks for your help!
@conda-forge-admin, please ping conda-forge/help-python

**Comment by speediedan** [comment]
@conda-forge/staged-recipes Just checking in on the review. Thanks for all your contributions to this invaluable community-driven project! Please let me know if there is anything else you need from me.

File: recipes/finetuning-scheduler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/finetuning-scheduler-{{ version }}.tar.gz
+  sha256: f073fb7cb89e7ba4efe9f79079ea36859424589fcb1a4e61c3b70f96d58996f1
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/finetuning-scheduler/meta.yaml:14]: This can probably be `noarch` instead. Can you check? Even if it fails Windows due to the lack of pytorch the noarch recipe is preferred.

Comment (speediedan) [reply recipes/finetuning-scheduler/meta.yaml]: Awesome, thanks so much @ocefpaf! I suspected that might be the case and had been planning to switch back to noarch in feedstock but I was trying to get all the checks to pass for this initial PR :facepalm: . 

I've gone ahead and switched to noaarch and the two expected builds are passing (macOS and Linux) with win_64 failing as expected. Let me know if there's anything else you need. Thanks again! :partying_face:
